src/sass/component_styles/_skillCard.scss

```diff
@@ -1,15 +1,23 @@
 .skill-card {
     color: $color-black;
-    padding: 2.5rem;
+    padding: 2.0rem;
     width: 25rem;
     border-radius: 5px;
     text-align: center;
     box-shadow: 0 1.5rem 4rem rgba(black, .15);
     background-color: $color-yellow;
-    transition: transform .3s;
+    transition: all .3s;
 
     &:hover {
         transform: translateY(-1.5rem) scale(1.05);
+
+        .card-title {
```

Comment:
When the skill card is hovered the headline enlarges, and the contents become bolder. This is done to make it easy to read! No one wants to strain to see ultra-fine fonts on a screen, and some users can't see it no matter how hard they squint.